test/schema/me/conversation/index.js

```diff
@@ -84,7 +84,7 @@ describe("Me", () => {
             metadata: {
               lewitt_invoice_id: "420i",
             },
-            from: "\"Percy Z\" <percy@cat.com>",
+            from: `"Percy Z" <percy@cat.com>`,
```

Comment:
interesting, I added these and was always having to commit with `--no-verify`  :( good to know about this!